lib/spack/spack/reporters/cdash.py

```diff
@@ -61,7 +62,7 @@ class CDash(Reporter):
     def __init__(self, args):
         Reporter.__init__(self, args)
         self.success = True
-        self.template_dir = os.path.join('reports', 'cdash')
+        self.template_dir = posixpath.join('reports', 'cdash')
```

Comment:
I'm curious if `posixpath` is still needed given https://github.com/spack/spack/pull/28402

Reply:
In both reporters, the posixpath is required (this is one of the very few cases of this) due to an internal use of `/` separators.

Reply:
As discussed and now commented, this is due to Jinja2's internal useage of '/'